### PR TITLE
select/clear all added

### PR DESCRIPTION
**Pull request by sprajjwal**
added the option to select unselect all the options on the page.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -107,9 +109,24 @@ class App extends React.Component {
...
+    // if (data === "all") {
+    //   for (const item in this.state) {
+    //     this.setState(function(state, props) {
+    //       return {
+    //         [item]: state["all"]
+    //       }
+    //     })
+    //   }
+    // } 
+  }
```
Comment (ytsheng) [inline src/App.js:124]: remove.

File: src/ExploreFunctions.js [hunk 2]
```diff
@@ -1,13 +1,31 @@
...
+
+// function checkAll(items) {
+//   for (const item in items) {
+//     if (items[item] === false) {
+//       console.log('false found')
+//       return false
+//     }
+//   }
+//   return true;
+// }
```
Comment (ytsheng) [inline src/ExploreFunctions.js:16]: please remove all your commented out code, we don't want commented code in production

File: src/ExploreFunctions.js [hunk 3]
```diff
@@ -1,13 +1,31 @@
...
 
 class ExploreFunctions extends React.Component {
   render() {
     return (
         <div>
-            <p className="ExploreFunctions-header">Explore Functions:</p>
```
Comment (ytsheng) [inline src/ExploreFunctions.js:9]: remove the spaces since the buttons are no longer added here.

File: src/App.js [hunk 4]
```diff
@@ -107,9 +109,24 @@ class App extends React.Component {
...
+
+  setAll(bool) {
```
Comment (ytsheng) [inline src/App.js:126]: rename the variable from bool to selected, bool in some languages are reserved.

**Comment by sprajjwal** [comment]
Removed bogus code/comments. Updated the variable name!
the select all button should select all tags and clear all button will clear all tags.

tnt

**Comment by ytsheng** [comment]
Cool, looks good to me. Thank you so much!

**Pull request merged**